docs/examples/offloading-multiple-pods.md

```diff
@@ -0,0 +1,251 @@
+# Offloading multiple pods
+
+In this tutorial you will learn how to deploy an application, and use liqo to replicate it on other clusters.
+
+![Offloading Multiple Pods Overview](/_static/images/examples/offloading-multiple-pods/offloading-multiple-pods.drawio.svg)
+
+More specifically, you will configure a scenario composed of a *single entry point cluster* leveraged for the deployment of the applications (called *origin cluster*) and two *destination clusters*.
+The deployed application will be replicated on every *destination cluster* in order to deploy exactly **one** identical *pod* on each *destination cluster*.
+
+## Provision the playground
+
+First, check that you are compliant with the [requirements](/examples/requirements.md).
+
+Then, let's open a terminal on your machine and launch the following script, which creates the three above-mentioned clusters with KinD and installs Liqo on all of them.
+
+{{ env.config.html_context.generate_clone_example('offloading-multiple-pods') }}
+
+Export the kubeconfigs environment variables to use them in the rest of the tutorial:
+
+```bash
+export KUBECONFIG=liqo_kubeconf_origin
+export KUBECONFIG_DESTINATION1=liqo_kubeconf_destination1
+export KUBECONFIG_DESTINATION2=liqo_kubeconf_destination2
```

Comment:
The other examples have cities' names for clusters. I would follow this pattern.

They can be something like `europe-cloud`, `europe-yourcity1-edge`, and `europe-yourcity2-edge` (choose 2 cities you like).